### ISSUE TITLE
Copy all playlists - no YTMusic playlists created

### ISSUE THREAD

**Issue by kandyman2017** [linsomniac/spotify_to_ytmusic#131]
I got to Tab 5 (copy all playlists) in the UI, and it generated a list of Spotify playlists, but no YT Music playlists are listed under it.
![image](https://github.com/user-attachments/assets/787d7caa-3f1e-4bce-ac18-b4996d79a686).

I let it run for a while but nothing happened. No new playlists appear in YT Music. Is there another step I am missing??

**Comment by Casudeus** [comment]
yeah the sames happening to me, ive been trying to reinstall things but the issue persists....can someone help??

**Comment by VladPav04** [comment]
Same for me. No youtube list displayed. Any action done regarding youtube playlists (list/create/copy) always returns the same thing: "Server returned HTTP 400: Bad Request". I have attached a stacktrace for the s2yt_copy_playlist command. 

[Stack trace for s2yt_copy_playlist.txt](https://github.com/user-attachments/files/17813694/Stack.trace.for.s2yt_copy_playlist.txt)

**Comment by Casudeus** [comment]
#132

**Comment by Kaidep13** [comment]
Hey man. I am not a coder so can you explain what i should do? How did you edit the code as you mentioned in the that link?

**Comment by Kaidep13** [comment]
Hey man. Thanks for this! If only i saw it a bit earlier. Did it in the hard way as mentioned in #125 and made it work
